From a5b708a5de0404cefe1ca38b985f315523515cb4 Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Sat, 19 Oct 2013 12:55:29 +0200
Subject: [PATCH] b

---
 init.rc | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/init.rc b/init.rc
index 1f635c9..ad6cc4f 100755
--- a/init.rc
+++ b/init.rc
@@ -75,6 +75,8 @@ loglevel 3
     mkdir /mnt/shell 0700 shell shell
     mkdir /mnt/media_rw 0700 media_rw media_rw
     mkdir /storage 0751 root sdcard_r
+    # Create mountpoint so Dalvik can mark as slave in zygotes.
+    mount tmpfs tmpfs /storage mode=0050,uid=0,gid=1028
 
     # Directory for putting things only root should see.
     mkdir /mnt/secure 0700 root root
@@ -610,3 +610,10 @@ service mdnsd /system/bin/mdnsd
     socket mdnsd stream 0660 mdnsr inet
     disabled
     oneshot
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class main
+    user root
+    group root
+    oneshot
-- 
1.8.1.2

